nova/conf/scheduler.py

```diff
@@ -517,6 +517,19 @@
 
 * Multiple integer keys mapping to integer or float values, where the value
   corresponds to a weight and the integer to an upper bound of memory.
+"""),
+    cfg.FloatOpt("decommissioning_weight_multiplier",
+                 default=1.0,
+                 help="""
+Multiplier used for de-prioritizing hosts which are under decommissioning.
```

Comment:
This should mention the exact trait that needs to be set. I know this will duplicate the string value instead of reusing `DECOM_TRAIT`, but I think it's still better than not mentioning it here too.